### ISSUE TITLE
<Sticky> component, two issues

### ISSUE THREAD

**Issue by yumikohey** [OfficeDev/office-ui-fabric-react#11325]
### Environment Information
All

### Describe the issue:
We use <Sticky> for fixing the column header on the top of <DetailsList />
1. When we switch layout between details list and compact list, two headers will show up when switching layout.

2. accessibility: With Sticky header, the select all radio button will be focused after selecting an item. It doesn't follow the focus order visually. 

### Please provide a reproduction of the issue in a codepen:
https://microsoft-my.sharepoint-df.com/:v:/p/huaxi/EcDalsQc9UhJtWTtEEj3BeYBN3CTL2csGoVjdFkXS6qXXQ?e=XaLtyO

#### Actual behavior:
1. It shows up two column headers when switching layout.
2. <Sticky /> component is always at the bottom of the DOM no matter is sticky on the top or sticky at the bottom.

#### Expected behavior:
1. It shouldn't show up two column headers when switching layout.
2. <Sticky /> component should be on the top of the list if it's sticky on the top and should be at the bottom of the list if it's sticky at the bottom.

**Comment by khmakoto** [comment]
Hi @yumikohey, thanks for filing the issue! Regarding the issues you're reporting:

1. What do you mean by two column headers showing up? I didn't see anything like this when checking the url you provided.
2. I think I know what you're referring to, but it's hard to notice in the video you sent.

In both cases, an isolated codepen I could fiddle with would be great to help me get some traction. Also, please do not post internal Microsoft URLs here, as this is an open source platform. Thanks!

**Comment by yumikohey** [comment]
@khmakoto How does it go?

**Comment by khmakoto** [comment]
Hi @yumikohey, sorry, I was very busy last week. I'm trying to replicate this issue in an isolated codepen so that I can identify what's causing this behavior but haven't been able to so far. I'll write an update here with my findings.

**Comment by yumikohey** [comment]
@khmakoto Thank you

**Comment by khmakoto** [comment]
Hi @yumikohey, I've been trying to replicate this behavior in an isolated codepen but have so far not been able to. Would you mind trying to replicate it in a codepen yourself so that I can start acting on it? Thanks!

**Comment by yumikohey** [comment]
@khmakoto  I just created a codepen that can repro the accessibility issue.
https://codepen.io/yumikohey/pen/PowbQay
Please let me know what's the ETA for the fix? Thank you so much.

**Comment by khmakoto** [comment]
Hi @yumikohey, I tried reproducing the first bug you mentioned in [this codepen](https://codepen.io/khmakoto/pen/NWPXEaR?editors=1010) but was unable to. Could you try changing it so that I can have an isolated repro of the bug? Thanks!
